File: server/node/config/querys/university_query.js
const university_query = {
    addUniversity: (university) => {
        return `INSERT INTO "public"."rs_universities" 
        ("university_Id", "university_name", "geoLocation_Id")
        VALUES 
        ('${university.university_Id}', '${university.university_name}', '${university.geoLocation_Id}');`;
    },
    updateUniversity: (university) => {
        return `UPDATE "public"."rs_universities" SET 
                    "university_name" = '${university.university_name}',
                    "geoLocation_Id" = '${university.geoLocation_Id}' 
                WHERE "university_Id" = '${university.university_Id}';`;
    },
    deleteUniversityById: (university) => {
        return `DELETE FROM "public"."rs_universities" 
        WHERE "university_Id" = '${university.university_Id}';`;
    },
    deleteUniversityByGeoLocationId: (university) => {
        return `DELETE FROM "public"."rs_universities" 
        WHERE "geoLocation_Id" = '${university.geoLocation_Id}';`;
    },
    getUniversityById: (university) => {
        return `SELECT * FROM "public"."rs_universities"
                WHERE "university_Id" = '${university.university_Id}';`;
    },
    getUniversityByGeoLocationId: (university) => {
        return `SELECT * FROM "public"."rs_universities"
                WHERE "geoLocation_Id" = '${university.geoLocation_Id}';`;
    },
    getAllUniversities: () => {
        return `SELECT * FROM "public"."rs_universities";`;
    },
}

module.exports = university_query;

File: server/java/src/main/java/controller/RDS/querys/university_query.java
package controller.RDS.querys;
import controller.RDS.models.University;

public class university_query {
    static public String addUniversity(University u) {
        return "INSERT INTO \"public\".\"rs_universities\" (\"university_Id\", \"geoLocation_Id\", \"university_name\") " +
                "VALUES " +
                "('"+u.getUniversityId()+"', '"+u.getGeoLocation_Id()+"', '"+u.getUniversity_name()+"');";
    }
    static public String updateUniversity(University u) {
        return "UPDATE \"public\".\"rs_universities\" SET " +
                "\"university_name\" = '"+u.getUniversity_name()+"', " +
                "\"geoLocation_Id\" = '"+u.getGeoLocation_Id()+"' " +
                "WHERE \"university_Id\" = '"+u.getUniversityId()+"';";
    }
    static public String deleteUniversityById(University u) {
        return "DELETE FROM \"public\".\"rs_universities\" WHERE \"university_Id\" = '"+u.getUniversityId()+"';";
    }
    static public String getUniversityById(University u) {
        return "SELECT * FROM \"public\".\"rs_universities\" WHERE \"geoLocation_Id\" = '"+u.getUniversityId()+"';";
    }
    static public String getAllUniversities() {
        return "SELECT * FROM \"public\".\"rs_universities\";";
    }
}
